docs/resources/aviatrix_edge_spoke_transit_attachment.md

```diff
@@ -19,6 +19,18 @@ resource "aviatrix_edge_spoke_transit_attachment" "test_attachment" {
   transit_gw_name = "transit-gw"
 }
 ```
+```hcl
+# Create an Aviatrix Edge as a Spoke to Edge as a Transit Attachment
+resource "aviatrix_edge_spoke_transit_attachment" "test_attachment_2" {
+  spoke_gw_name   = "e2e-edge-spoke-1"
+  transit_gw_name = "test-edge-transit-1"
+  enable_over_private_network = true
+  enable_jumbo_frame = false
+  enable_insane_mode = true
+  dst_wan_interfaces = "eth1"
```

Comment:
If the previous comment regarding the data type of `dst_wan_interfaces` is addressed, be sure to update the doc description and examples

Reply:
updated the doc